client/app/scripts/superdesk-desks/aggregate-widget/aggregate.js

```diff
@@ -15,7 +15,8 @@ angular.module('superdesk.aggregate.widgets', ['superdesk.aggregate', 'superdesk
                 thumbnail: 'scripts/superdesk-desks/aggregate-widget/thumbnail.svg',
                 template: 'scripts/superdesk-desks/aggregate-widget/aggregate-widget.html',
                 configurationTemplate: 'scripts/superdesk-desks/aggregate-widget/configuration.html',
-                description: 'Displaying the monitoring content view.',
+                description: 'This widget allows you to create literally any content view you may need in Superdesk, be it production or ingest.' +
+                        'All you need is to select a desk, its stages or a saved search. Name your view once you are done. Enjoy!',
```

Comment:
Do we really want to have the `"Enjoy!"` bit in the enterprise software?
(personally I don't mind it at all, but someone else might :) )